octane/evmengine/keeper/db.go

```diff
@@ -113,3 +114,34 @@ func (k *Keeper) listWithdrawalsByAddress(ctx context.Context, withdrawalAddr co
 
 	return withdrawals, nil
 }
+
+// ListWithdrawalsBelowHeight returns all withdrawals created below the specified height, sorted by the created height, limited by the provided count.
+func (k *Keeper) ListWithdrawalsBelowHeight(ctx context.Context, height uint64, limit int) ([]*Withdrawal, error) {
```

Comment:
why not `EligibleWithdrawals` that returns the thing we need exactly?

Reply:
Sure, we can go with that name as well.